AntWars/AI/AIAntBase.cs

```diff
@@ -35,11 +37,12 @@ public abstract class AIAntBase : IAIAnt {
         public abstract void notify(HashSet<Coordinates> coords);
 
         /// <summary>
-        /// Initialisiert eine neue Instanz der Random-Klasse unter Verwendung eines Seeds.
+        /// Gibt eine Instanz der Klasse Random zurück.
         /// </summary>
         /// <returns>Instanz der Random-Klasse</returns>
-        public Random getRandom() {
-            return new Random(Guid.NewGuid().GetHashCode());
+        public Random getRandom()
+        {
```

Comment:
formatting?, du kannst zwar selbst mit vs2013 die datei nicht öffnen aber du kannst da auch einstellungen drin machen das die klammern mit in der methoden deklarierung sind.